### DISCUSSION TITLE
Enterprise support for OpenFGA in production

### DISCUSSION THREAD

**Discussion by Singha2** [openfga/community#11] [General]
Hi Auth0 Team, is there any enterprise support model available for OpenFGA ? I have been following Auth0 - FGA SaaS since quite sometime, it seems OpenFGA has most of its feature other than UI. FGA-SaaS is available only under developer preview mode, while OpenFGA can be hosted on-prem and used in production by any enterprise. I am very much interested to know if there is any support model available for OpenFGA from Auth0 team.

**Comment by aaguiarz** [comment]
Hi @Singha2 

We are not currently planning to offer enterprise support for OpenFGA, but we've heard that request from customers so it's something we are considering.

We plan to enable customers to use Auth0 FGA in production around the end of October.

**Comment by AlessandroMinoccheri** [comment]
Hello, do you have any plan for enterprise support now? 
We would like to explore openFGA and use it in production but without the right support it can create problems.

Please, let me know @aaguiarz
